rsconnect_jupyter/__init__.py

```diff
@@ -53,9 +53,10 @@ def post(self, action):
         if action == 'verify_server':
             server_address = data['server_address']
             api_key = data['api_key']
+            disable_tls_check = data.get('disable_tls_check', False)
```

Comment:
In general, it might be better to require this key since the frontend should always be providing it.